### ISSUE TITLE
Apply Dropout to this model

### ISSUE THREAD

**Issue by alexw92** [hellochick/ICNet-tensorflow#67]
First of all, thank you @hellochick  for sharing this great code :+1: 

I wonder how to achieve better train results and I noticed that the dropout()- code from network.py
is actually never used in model. Didn't you find dropout necessary for this model or is there any other reason (bad results or implementation details) why you  didn't use it?

**Comment by hellochick** [comment]
Hey @alexw92 ,

Sorry for my late reply, thanks for your appreciation. I didn't use the dropout layer since I tried to re-produced the setting in the original paper. But you can try to add dropout layer when you find there exists overfitting problems in your own dataset!

**Comment by alexw92** [comment]
Thanks for your response @hellochick !
I agree with you and the authors. Dropout is not necessary in this model and I was able to
fix my overfitting issues by just applying validation during training
